ouroboros-network/src/Ouroboros/Network/Protocol/LocalTxSubmission/Client.hs

```diff
@@ -0,0 +1,69 @@
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE RecordWildCards     #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE RankNTypes          #-}
+
+
+-- | A view of the transaction submission protocol from the point of view of
+-- the client.
+--
+-- This provides a view that uses less complex types and should be easier to
+-- use than the underlying typed protocol itself.
+--
+-- For execution, 'localTxSubmissionClientPeer' is provided for conversion
+-- into the typed protocol.
+--
+module Ouroboros.Network.Protocol.LocalTxSubmission.Client (
```

Comment:
Mabe call this module `LocalClient`, this way we could keep the node2client and node2node clients in separate modules. As these modules are small, and they have just a few imports there's very little price to pay for splitting them.

Reply:
Hmm, I'm not so sure. The sets of protocols are not disjoint between the two bundles.